pkgs/servers/computing/slurm/default.nix

```diff
@@ -2,10 +2,20 @@
 , python, munge, perl, pam, openssl, zlib
 , ncurses, libmysqlclient, gtk2, lua, hwloc, numactl
 , readline, freeipmi, libssh2, xorg, lz4
+, buildEnv, libibmad, libibumad, pmix, ucx
 # enable internal X11 support via libssh2
 , enableX11 ? true
 }:
 
+let
+
+  ofed = buildEnv {
```

Comment:
`libibmad` and `libibumad` should be included in `rdma-core` now. Those packages are outdated.

Reply:
thought that too, but it didn't work for some reason. will test it again.

Reply:
It would be good if that works. Eventually `libibmad` and `libibumad` should be removed from nixpkgs.